pkg/pillar/dpcreconciler/linuxitems/bond.go

```diff
@@ -89,28 +90,204 @@ type BondConfigurator struct {
 	NetworkMonitor netmonitor.NetworkMonitor
 }
 
-// Create is not yet implemented.
+// Create adds new Bond interface.
 func (c *BondConfigurator) Create(ctx context.Context, item depgraph.Item) error {
-	// TODO
-	return errors.New("not implemented")
+	bondCfg := item.(Bond)
+	bond := netlink.NewLinkBond(netlink.LinkAttrs{Name: bondCfg.IfName})
+	switch bondCfg.Mode {
+	case types.BondModeUnspecified:
```

Comment:
Would it make sense to do this for default and unspecified so that we can detect if a new enum value is added and this code is not updated?